experiments/omip_prototype/download_data.jl

```diff
@@ -0,0 +1,8 @@
+using ClimaOcean
+using ClimaOcean.JRA55
+using ClimaOcean.DataWrangling: download_dataset
+
+atmosphere = JRA55PrescribedAtmosphere(; dir="forcing_data/", 
```

Comment:
Trying to encourage people not to set this variable because it makes setups less portable (eg data will be re-downloaded everywhere)